examples/features/standard/spring-boot-integration/pom.xml

```diff
@@ -0,0 +1,94 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.apache.activemq.examples.broker</groupId>
+		<artifactId>jms-examples</artifactId>
+		<version>2.7.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-boot-integration</artifactId>
+	<name>ActiveMQ Artemis JMS Spring Boot Integration Example</name>
+
+	<properties>
+		<activemq.basedir>${project.basedir}/../../../..</activemq.basedir>
+
+		<!-- AMQP Spring Starter -->
+		<amqp-10-jms-spring.version>2.0.1</amqp-10-jms-spring.version>
+		<!-- JMS Pool -->
+		<jms-pool.version>1.0.2</jms-pool.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.activemq</groupId>
+			<artifactId>artemis-amqp-protocol</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.amqphub.spring</groupId>
+			<artifactId>amqp-10-jms-spring-boot-starter</artifactId>
+			<version>${amqp-10-jms-spring.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.messaginghub</groupId>
```

Comment:
This does seem like If this is a spring boot example, and example using Spring JMS CachingConnection Factory would be more suitable. As spring users use this.